moveit_core/planning_request_adapter/include/moveit/planning_request_adapter/planning_request_adapter.h

```diff
@@ -63,6 +63,11 @@ class PlanningRequestAdapter
   {
   }
 
+  /// Initialize parameters using the passed NodeHandle
+  virtual void initialize(const ros::NodeHandle& node_handle)
+  {
```

Comment:
This default implementation should issue a warning indicating that the old way of initializing from a private `NodeHandle` is deprecated. In future this method should become abstract. Please leave a corresponding TODO.